### ISSUE TITLE
CTL: Bad game system trap: 582

### ISSUE THREAD

**Issue by codemasher** [PadWorld-Entertainment/worldofpadman#13]
The above error occurs when trying to start a CTL game. Build: https://github.com/PadWorld-Entertainment/wop-engine/commit/471e52f5c1fe1fcd7387c164e2d427fad7667713 , Tested on Windows, Linux.

Update: it does not occur with some community CTL maps, for example [Hectic's "Hurry up!"](http://www.zock-server.de/downloads/WoP_1.6/pk3-files/wop/hec_hurryup.pk3), so it's probably a map script issue?

**Comment by mgerhardy** [comment]
What was the map where the error occurred?

**Comment by codemasher** [comment]
It occurs with every included CTL map, it didn't crash when i started a game with Hectic's map.

**Comment by mgerhardy** [comment]
`trap_BotLibSourceFileAndLine` is missing - maybe a merge artifact. Will check it.

**Comment by mgerhardy** [comment]
which compiler(s) did Hectic use? Do we have that information? Where is that map? Do we have it?

**Comment by kai-li-wop** [comment]
This map is custom map by Hectic. It is not depending on the map. I also just tried to start a CTL dedicated server and a local one directly out of the client and both cause the same issue when starting e.g. PadAsia or any other CTL map. All maps were compiled with q3map2 2.5.16 back then. The error is a little cryptic to me. If I remember correct, we did some minor adjustment to the player team spawn points compared to q3, but don't as k me what it was exactly. Maybe just some wording of names. Could it be connected?
